hdlConvertor/_hdlConvertor.pyx

```diff
@@ -39,7 +39,7 @@ cdef int raise_my_py_error() except * :
     raise ParseException(msg)
 
 cdef extern from "hdlConvertor/convertor.h" namespace "hdlConvertor":
-    cdef cppclass Convertor:
+    cdef cppclass Convertor nogil:
```

Comment:
Well, this is an example what could happen if I do not read commits properly. I was just getting random crashes from very weird reasons.

Reply:
Agree.
This line should not have been committed. It was unintentional. 
Sorry about that...

On python 2.7@master I do not observe any crash... 
Are you able to provide additional details ?

Reply:
The error happens when the syntax error exception is raised somehow, but it is odd, because call to HdlConvertor.parse is blocking and the app is single threaded. It was happening on large source files only.